src/__tests__/db.integration.test.js

```diff
@@ -1,115 +1,41 @@
 import { v4 as uuid } from 'uuid';
 import ModelFactory from '../models';
-import { modelName as messageFragmentModel } from '../models/message-fragment';
 import { modelName as registrationRequestModel } from '../models/registration-request';
-import { modelName as messageIdReplacementModel } from "../models/message-id-replacement";
-import { readFile } from './utilities/integration-test.utilities';
-import {
-  updateFragmentMessageId
-} from '../services/transfer/transfer-out-util';
-import {
-  getFragment,
-  retrieveIdsFromEhrRepo
-} from '../services/ehr-repo/get-fragment';
-import { transferOutFragments } from '../services/transfer/transfer-out-fragments';
-import { createRegistrationRequest } from '../services/database/create-registration-request';
-import { logger } from "../config/logging";
-import { createMessageIdReplacement } from "../services/database/create-message-id-replacement";
-import nock from "nock";
-import { config } from "../config";
-
-jest.mock('../services/ehr-repo/get-fragment');
-jest.mock('../services/transfer/transfer-out-util');
+import { createRegistrationRequest } from "../services/database/create-registration-request";
+import {logInfo} from "../middleware/logging";
```

Comment:
```suggestion
import { logInfo } from "../middleware/logging";
```
Space between the curly brackets and variable name for consistency and readabilitiy